sdk/core/azure-core/src/main/java/com/azure/core/util/serializer/JacksonAdapter.java

```
@@ -59,6 +60,49 @@
+    private static final MethodHandle SET_COERCION;
+    private static final Object COERCION_INPUT_SHAPE_EMPTY_STRING;
+    private static final Object COERCION_ACTION_AS_NULL;
+    private static final boolean USE_REFLECTION_TO_SET_COERCION;
+
+    static {
+        MethodHandles.Lookup publicLookup = MethodHandles.publicLookup();
+
+        MethodHandle coercionConfigDefaults = null;
+        MethodHandle setCoercion = null;
+        Object coercionInputShapeEmptyString = null;
+        Object coercionActionAsNull = null;
+        boolean useReflectionToSetCoercion = false;
+
+        try {
+            Class<?> mutableCoercionConfig = Class.forName(MUTABLE_COERCION_CONFIG);
+            Class<?> coercionInputShapeClass = Class.forName(COERCION_INPUT_SHAPE);
+            Class<?> coercionActionClass = Class.forName(COERCION_ACTION);
+
+            coercionConfigDefaults = publicLookup.findVirtual(ObjectMapper.class, "coercionConfigDefaults",
+                MethodType.methodType(mutableCoercionConfig));
+            setCoercion = publicLookup.findVirtual(mutableCoercionConfig, "setCoercion",
+                MethodType.methodType(mutableCoercionConfig, coercionInputShapeClass, coercionActionClass));
+            coercionInputShapeEmptyString = publicLookup.findStaticGetter(coercionInputShapeClass, "EmptyString",
+                coercionInputShapeClass).invoke();
+            coercionActionAsNull = publicLookup.findStaticGetter(coercionActionClass, "AsNull", coercionActionClass)
+                .invoke();
+            useReflectionToSetCoercion = true;
+        } catch (Throwable ignored) {
+        }
+
+        COERCION_CONFIG_DEFAULTS = coercionConfigDefaults;
+        SET_COERCION = setCoercion;
+        COERCION_INPUT_SHAPE_EMPTY_STRING = coercionInputShapeEmptyString;
+        COERCION_ACTION_AS_NULL = coercionActionAsNull;
+        USE_REFLECTION_TO_SET_COERCION = useReflectionToSetCoercion;
```

Same comment as above - better to throw an exception.